src/Spatial/Units/Angle.cs

```diff
@@ -13,6 +13,21 @@ namespace MathNet.Spatial.Units
     [Serializable]
     public struct Angle : IComparable<Angle>, IEquatable<Angle>, IFormattable, IXmlSerializable
     {
+        /// <summary>
```

Comment:
No Zero?

Reply:
I started with the angles used in common geometry calculations.  I don't know that I have ever come across using a zero angle for anything, but why not

Reply:
Arguably `Degrees0` counts as zero